src/providers/formattingProvider.ts

```diff
@@ -209,8 +209,11 @@ export class FormatProvider implements vscode.DocumentFormattingEditProvider {
                 depth++;
             }
 
-            // default or hotkey
-            if (!moreOpenParens && purifiedLine.match(/:\s*$/)) {
+            // default or hotkey or label
+            if (
+                !moreOpenParens &&
+                purifiedLine.match(/^(?:\s*[^\s\t,`]+):\s*$/) // Label name may consist of any characters other than space, tab, comma and the escape character (`)
```

Comment:
Same as https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/pull/163#discussion_r922964092, let's add some unit tests to make this regex a bit more readable.

Reply:
Done.